Fail to LoadPolicy

I use [casbin-pg-adapter](https://github.com/casbin/casbin-pg-adapter/tree/v1.0.3) , have an error msg.

**this is  error msg:** 

```
PS D:\Documents\test> go run .\main.go
2021/11/29 12:42:34 错误 #42703 字段 casbin_rule.p_type 不存在
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x10 pc=0x8b110d]

goroutine 1 [running]:
github.com/casbin/casbin/v2.(*Enforcer).LoadPolicy(0x0)
        C:/Users/Administrator/go/pkg/mod/github.com/casbin/casbin/v2@v2.39.1/enforcer.go:284 +0x4d
main.main()
        D:/Documents/test/main.go:24 +0x105
exit status 2
```

when i change the field `ptype` to `p_type`, it's ok.

```
PS D:\Documents\test> go run .\main.go
2021/11/29 12:52:49 true
```

**this is my enviorment:**

```
go 1.17.2
casbin v2.39.1
casbin-pg-adapter v1.0.3
PostgreSQL 12.8
```

**this is my code:**

```go

package main

import (
	"log"

	pgadapter "github.com/casbin/casbin-pg-adapter"
	"github.com/casbin/casbin/v2"
)

func main() {

	url := "postgresql://postgres:111111@localhost:5432/casbin?sslmode=disable"

	a, err := pgadapter.NewAdapter(url)
	if err != nil {
		log.Println(err)
	}
	e, err := casbin.NewEnforcer("model.conf", a)
	if err != nil {
		log.Println(err)
	}

	// Load the policy from DB.
	err = e.LoadPolicy()
	if err != nil {
		log.Println(err)
	}

	// Check the permission.
	b1, err := e.Enforce("alice", "data1", "write")
	if err != nil {
		log.Println(err)
	}
	log.Println(b1)

	// Save the policy back to DB.
	e.SavePolicy()
}
```

--- a/adapter.go
+++ b/adapter.go
@@ -18,7 +18,7 @@ const DefaultTableName = "casbin_rule"
 type CasbinRule struct {
 	tableName struct{} `pg:"_"`
 	ID        string
-	PType     string
+	Ptype     string
 	V0        string
 	V1        string
 	V2        string
@@ -142,12 +142,12 @@ func (r *CasbinRule) String() string {
 	var sb strings.Builder
 
 	sb.Grow(
-		len(r.PType) +
+		len(r.Ptype) +
 			len(r.V0) + len(r.V1) + len(r.V2) +
 			len(r.V3) + len(r.V4) + len(r.V5),
 	)
 
-	sb.WriteString(r.PType)
+	sb.WriteString(r.Ptype)
 	if len(r.V0) > 0 {
 		sb.WriteString(prefixLine)
 		sb.WriteString(r.V0)
@@ -200,7 +200,7 @@ func policyID(ptype string, rule []string) string {
 }
 
 func savePolicyLine(ptype string, rule []string) *CasbinRule {
-	line := &CasbinRule{PType: ptype}
+	line := &CasbinRule{Ptype: ptype}
 
 	l := len(rule)
 	if l > 0 {
@@ -337,7 +337,7 @@ func (a *Adapter) RemovePolicies(sec string, ptype string, rules [][]string) err
 
 // RemoveFilteredPolicy removes policy rules that match the filter from the storage.
 func (a *Adapter) RemoveFilteredPolicy(sec string, ptype string, fieldIndex int, fieldValues ...string) error {
-	query := a.db.Model((*CasbinRule)(nil)).Table(a.tableName).Where("p_type = ?", ptype)
+	query := a.db.Model((*CasbinRule)(nil)).Table(a.tableName).Where("ptype = ?", ptype)
 
 	idx := fieldIndex + len(fieldValues)
 	if fieldIndex <= 0 && idx > 0 && fieldValues[0-fieldIndex] != "" {
@@ -413,7 +413,7 @@ func (a *Adapter) loadFilteredPolicy(model model.Model, filter *Filter, handler 
 	if filter.P != nil {
 		lines := []*CasbinRule{}
 
-		query := a.db.Model(&lines).Table(a.tableName).Where("p_type = 'p'")
+		query := a.db.Model(&lines).Table(a.tableName).Where("ptype = 'p'")
 		query, err := buildQuery(query, filter.P)
 		if err != nil {
 			return err
@@ -430,7 +430,7 @@ func (a *Adapter) loadFilteredPolicy(model model.Model, filter *Filter, handler 
 	if filter.G != nil {
 		lines := []*CasbinRule{}
 
-		query := a.db.Model(&lines).Table(a.tableName).Where("p_type = 'g'")
+		query := a.db.Model(&lines).Table(a.tableName).Where("ptype = 'g'")
 		query, err := buildQuery(query, filter.G)
 		if err != nil {
 			return err
@@ -474,7 +474,7 @@ func (a *Adapter) UpdatePolicies(sec string, ptype string, oldRules, newRules []
 func (a *Adapter) UpdateFilteredPolicies(sec string, ptype string, newPolicies [][]string, fieldIndex int, fieldValues ...string) ([][]string, error) {
 	line := &CasbinRule{}
 
-	line.PType = ptype
+	line.Ptype = ptype
 	if fieldIndex <= 0 && 0 < fieldIndex+len(fieldValues) {
 		line.V0 = fieldValues[0-fieldIndex]
 	}
@@ -529,9 +529,9 @@ func (a *Adapter) UpdateFilteredPolicies(sec string, ptype string, newPolicies [
 }
 
 func (c *CasbinRule) queryString() (string, []interface{}) {
-	queryArgs := []interface{}{c.PType}
+	queryArgs := []interface{}{c.Ptype}
 
-	queryStr := "p_type = ?"
+	queryStr := "ptype = ?"
 	if c.V0 != "" {
 		queryStr += " and v0 = ?"
 		queryArgs = append(queryArgs, c.V0)
@@ -562,8 +562,8 @@ func (c *CasbinRule) queryString() (string, []interface{}) {
 
 func (c *CasbinRule) toStringPolicy() []string {
 	policy := make([]string, 0)
-	if c.PType != "" {
-		policy = append(policy, c.PType)
+	if c.Ptype != "" {
+		policy = append(policy, c.Ptype)
 	}
 	if c.V0 != "" {
 		policy = append(policy, c.V0)

